### ISSUE TITLE
irregular result for prediction for identical object

### ISSUE THREAD

**Issue by DevLob-zz** [tensorflow/models#8207]
I used pre-trained model faster_rcnn_resnet101_coco.config with my own dataset

i have two issues 1- some elements were missed wile i learned it with high number of steps and test over the same learning data but still have some missing data

2- in sometimes same identical objects predicted correctly on some images and sometimes it got missed although the objects is identical

any help would be appreciated

tensorflow prediction

**Comment by Eshan-Agarwal** [comment]
When you train you model using some training data there is some accuracy you achieve let say 99% so when you try to test your model on same training data you there is chance of getting wrong predictions for 1 % of images. Can you please define your problem more accurate what is higher number of steps don't get it.

**Comment by DevLob-zz** [comment]
i have an image let say have two identical cat it can predict the one of them with low score 10% and not predict the other image just the position is different 
 i reach more than 200K step even this happened also with a model build for 1.1M steps 

some missing elements also i got total logloss very low values around 0.01

when i change the 
first_stage_max_proposals: default value to higher number this solve some of objectness elements
